_data/team.yml

```diff
@@ -130,6 +130,13 @@ people:
   pic: ali_raza_zaidi.jpg
   position: University of Waterloo Software Developer Co-op, UHN
   team: implementation
+  
+  - name: Niharika Srivatsa
+  lastname:  Srivatsa
```

Comment:
Spaces are significant in YAML files - you'll have to indent the fields 2 spaces so (_e.g._) `lastname` is lined up with `name`